tests/import_test.go

```diff
@@ -135,6 +135,29 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 			Expect(f.GetDiskGroup(f.Namespace, pvc, false)).To(Equal("107"))
 		}
 	})
+
+	It("Import registry disk into PVC barely big enough", func() {
+		imageURL := fmt.Sprintf(utils.LargePhysicalDiskQcow, f.CdiInstallNs)
+
+		By(imageURL)
+		dv := utils.NewDataVolumeWithHTTPImport("large-import", "2.25Gi", imageURL)
```

Comment:
The description of this PR mentions registry import, while here we are doing an http import.